src/app/app-client.ts

```diff
@@ -1175,4 +1184,104 @@ export class AppClient {
   ): Promise<CreateKeyFromExistingKeyAuthorizationsResponse> {
     return this.client.createKeyFromExistingKeyAuthorizations({ id });
   }
+
+  /**
+   * Retrieves user-defined metadata for an organization.
+   *
+   * @param id The ID of the organization
+   * @returns The metadata associated with the organization
+   */
+  async getOrganizationMetadata(
+    id: string
+  ): Promise<GetOrganizationMetadataResponse> {
+    return this.client.getOrganizationMetadata({ organizationId: id });
+  }
+
+  /**
+   * Updates user-defined metadata for an organization.
+   *
+   * @param id The ID of the organization
+   * @param data The metadata to update
+   * @returns Response indicating success or failure
```

Comment:
I think we should get rid of this and just make this a `void`-returning method. The response doesn't have anything in it so it doesn't actually tell the user anything useful, but that isn't necessarily obvious to a user so they might think that there's like a `success` field in the response or something. And in case of a failure, there would be a panic or raised error, so the `response` wouldn't be received.